api/public/cerbos/request/v1/request.proto

```diff
@@ -523,6 +523,28 @@ message ListPoliciesRequest {
   ];
 }
 
+message FilterPoliciesRequest {
+  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+    json_schema: {description: "Filter policies request"}
+  };
+  bool include_disabled = 1 [
+    (google.api.field_behavior) = OPTIONAL,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Include disabled policies"}
+  ];
+  string name_regexp = 2 [
```

Comment:
[I suggested introducing a new RPC](https://github.com/cerbos/cerbos/issues/1609#issuecomment-1582252664) because filtering can only be done on database-backed stores at the moment. My reasoning was that because the `ListPolicies` RPC works on all stores, it might be confusing if the filter option did not work for some stores.

Reply:
I feel like in terms of documentation, adding a caveat to the request fields about the supported stores is similar to adding one to an RPC itself, and it leaves the door open for us to implement this functionality for other stores without having to keep the two RPCs for backwards compatibility?

Reply:
Yeah that's an interesting one. We briefly discussed this [here](https://github.com/cerbos/cerbos/issues/1609#issuecomment-1582252664). We already support `ListPolicies` for non-mutable stores, but not other admin specific RPCs, so it might be cleaner to separate this mutable-only interface?

Reply:
Oh I didn't refresh. You guys type fast AF

Reply:
I am torn on this. Both options have their merits. Haines' point about being able to implement the functionality to immutable stores at a later date without changing anything is compelling though. We probably don't want to end up with two almost identical RPCs in the end. So, I think I'll vote for adding the functionality to `ListPolicies`. Sorry about the confusion.